docs/rc.local

```diff
@@ -0,0 +1,95 @@
+#!/bin/sh
+
+DIR=/opt/cohda/application/rsu1609
```

Comment:
Are these arguments (DIR, RSU) expected to be the same for all the tested RSUs - Cohda (MK5/MK6C/MK6), Commsignia, Kapsch?